File: app/src/main/java/mydebts/android/app/data/db/ParticipantContract.java
package mydebts.android.app.data.db;

import android.provider.BaseColumns;

public interface ParticipantContract extends BaseColumns {
    String TABLE_NAME = "Participants";
    String COLUMN_PERSON_ID = "person_id";
    String COLUMN_EVENT_ID = "event_id";
    String COLUMN_DEBT = "debt";
}

File: app/src/main/java/mydebts/android/app/data/db/ParticipantContract.kt
package mydebts.android.app.data.db

interface ParticipantContract {
    companion object {
        const val TABLE_NAME = "Participants"
        const val COLUMN_ID = "_id"
        const val COLUMN_PERSON_ID = "person_id"
        const val COLUMN_EVENT_ID = "event_id"
        const val COLUMN_DEBT = "debt"
    }
}
